src/index.js

```diff
@@ -11,7 +11,12 @@ import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import Glean from '@mozilla/glean/web';
 
+// Initialize Glean. ToDo: Replace uploadEnabled: true with a user preference
```

Comment:
@abhi-agg where is the data review request for this? If you had conversations with data stewardship on how to handle consent on this one, please record it on the related bug. Let's not merge this until this is recorded.

Reply:
@Dexterp37 I haven't filed a data review request for this one yet. @rosahbruno recommended that we land the integration first to find out exactly the metrics that we want to collect so that we don't have to keep submitting data requests again and again.

The next PR that I will be submitting will be about handling the data collection consent using [DNT](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack) flag. I had discussions with the data steward regarding this approach but I haven't recorded it in the [corresponding bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1851456). I will do it right away.

Please let me know if I misunderstood something 👍🏾

Reply:
> @Dexterp37 I haven't filed a data review request for this one yet. @rosahbruno recommended that we land the integration first to find out exactly the metrics that we want to collect so that we don't have to keep submitting data requests again and again.

Note that, [as per our integration checklist](https://mozilla.github.io/glean/book/user/adding-glean-to-your-project/index.html) (step 5), you need a data-review to add glean to the application **and** a data-review for the metrics. If @travis79 is fine with us landing this without, I'm fine with it as well, but let's record any decision around this on bugzilla.

> The next PR that I will be submitting will be about handling the data collection consent using [DNT](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack) flag. I had discussions with the data steward regarding this approach but I haven't recorded it in the [corresponding bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1851456). I will do it right away.
> 
> Please let me know if I misunderstood something 👍🏾

Works for me, please record the decision in the bug.

Reply:
As per our discussion on matrix, I:
1. Disabled the telemetry in this PR (`uploadEnabled: false` in `Glean.initialize` call)
2. Added a [comment in the data review bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1851456#c1) regarding handling the data collection consent.